venv_template.mk

```diff
@@ -3,11 +3,11 @@ venv.$(1): venv.$(1)/bin/activate venv.$(1)/bin/environment.sh
 
 venv.$(1)/bin/activate:
 	@echo "Creating $(1) venv in $$@ from $$<"
-	python3.10 -m venv venv.$(1)
+	python3.12 -m venv venv.$(1)
```

Comment:
was it really required? moreover, it belongs to the commit that does not explain it